config/environments/production.rb

```diff
@@ -104,4 +105,14 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  config.paperclip_defaults = {
+    storage: s3,
+    s3_credentials: {
+      bucket: ENV.fetch('S3_BUCKET_NAME'),
+      access_key_id: ENV.fetch('AWS_ACCESS_KEY_ID'),
+      secret_access_key: ENV.fetch('AWS_SECRET_ACCESS_KEY'),
+      s3_region: ENV.FETCH('AWS_REGION'),
```

Comment:
I believe bucket here should be “AWS_BUCKET” and region should be “AWS_BUCKET_REGION,” as in the last commit!